src/Features/Core/Portable/FindUsages/ExternalReferenceItem.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.FindUsages
+{
+    /// <summary>
+    /// Information about a symbol's reference that can be used for display and navigation in an
+    /// editor.  These generally reference items outside of the Roslyn <see cref="Solution"/> model
+    /// provided by external sources (for example: RichNav).
```

Comment:
Rather than sticking "RichNav" in the codebase, do we have some more understandable term? I'm reminded of how many complaints we get for references to "Venus" that are indecipherable if you aren't aware of history.

Reply:
maybe we can just call it "external sources such as online indexes"